spec/lib/request_button_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require 'request_button'
+
+describe RequestButton do
+  # Used to test the request button based on a specific version, URL, and accept header.
+  #
+  # Example:
+  # context 'when version is :version' do
+  #   include_examples(
+  #     'generates the button HTML based off of the provided parameters',
+  #     :version,
+  #     version_url,
+  #     version_header
+  #   )
+  # end
+  #
```

Comment:
nit: remove this line.

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/pull/429/commits/d214b95f8e62ce5520249d811393fbffa4b88df2#diff-73b0126b1ae7913430e9706705fca78aR16-R17